src/webhooks/registry.ts

```diff
@@ -336,7 +341,8 @@ const WebhooksRegistry: RegistryInterface = {
           shop,
           deliveryMethod,
         };
-        const returnedRegister: RegisterReturn = await WebhooksRegistry.register(webhook);
+        const returnedRegister: RegisterReturn =
+          await WebhooksRegistry.register(webhook);
```

Comment:
TypeScript understands that `await Promise<T>` returns a `T`, so this should be able to depend on type inference
```suggestion
        const returnedRegister = await WebhooksRegistry.register(webhook);
```